### PR TITLE
Updated to v3 version, supports image generated videos, with a maximum support of 960x960x144 video generation. 

### PR DESCRIPTION
**Pull request by bubbliiiing**

### CODE REVIEW

File: easyanimate/data/dataset_image_video.py
Comment (yunkchen) [inline easyanimate/data/dataset_image_video.py:1]: https://github.com/aigc-apps/EasyAnimate/issues/41

**Review by yunkchen — APPROVED** [review]

**Comment by wangqiang9** [comment]
Hi, will there be code update for image-to-video in this PR?

**Comment by bubbliiiing** [comment]
> Hi, will there be code update for image-to-video in this PR?

Yes

**Comment by wangqiang9** [comment]
Hi, will there be training code `scripts/train_i2v.py` update for image-to-video in this PR?

**Comment by bubbliiiing** [comment]
> Hi, will there be training code `scripts/train_i2v.py` update for image-to-video in this PR?

The image to video code is in scripts/train_t2iv.py.

**Comment by bubbliiiing** [comment]
> Hi, will there be training code `scripts/train_i2v.py` update for image-to-video in this PR?

The image to video code is in scripts/train_t2iv.py.

**Comment by bubbliiiing** [comment]
> Hi, will there be training code `scripts/train_i2v.py` update for image-to-video in this PR?

The image to video code is in scripts/train_t2iv.py.

**Pull request merged**